ipi/inputs/thermostats.py

```diff
@@ -98,6 +99,15 @@ class InputThermoBase(Input):
                 "help": "Scaling for the PILE damping relative to the critical damping. (gamma_k=2*lambda*omega_k",
             },
         ),
+        "pile_centroid_t": (
+            InputValue,
+            {
+                "dtype": float,
+                "default": 0.0,
+                "help": "Option to set a different centroid temperature wrt. that of the ensemble. Only used if value other than 0.0.",
```

Comment:
This helpline could be more useful, perhaps mention that this is useful for PIGS

Reply:
Regarding the "use in PIGS", I would not add that line @ceriottm -- this is because I have indeed extensively benchmarked it for two models and the realm of "accuracy" is still open. However this should not stop one from adding this functionality to the code because it is indeed a very small change and useful in general (maybe even for different applications).

Reply:
(but I will make the help string more helpful)